### PR TITLE
[stable/openebs] Update charts to 1.2.0 release

### PR DESCRIPTION
**Pull request by ChandanSagar**
- Updated README
- Updated chart version 1.2.0
- Updated values.yaml
- Updated ENV variales for openebs apiserver

Signed-off-by: ChandanSagar <chandan.pradhan@mayadata.io>

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/openebs/templates/validationwebhook.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ metadata:
 webhooks:
 # failurePolicy Fail means that an error calling the webhook causes the admission to fail.
   - name: admission-webhook.openebs.io
-    failurePolicy: {{ .Values.webhook.failurePolicy }}
```
Comment (prateekpandey14) [inline stable/openebs/templates/validationwebhook.yaml]: this change is not required.

File: stable/openebs/templates/deployment-ndm-operator.yaml [hunk 2]
```diff
@@ -56,10 +58,10 @@ spec:
         - name: CLEANUP_JOB_IMAGE
           value: "{{ .Values.ndmOperator.cleanupImage }}:{{ .Values.ndmOperator.cleanupImageTag }}"
 {{- if .Values.ndmOperator.nodeSelector }}
-      nodeSelector:
+     nodeSelector:
```
Comment (kmova) [inline stable/openebs/templates/deployment-ndm-operator.yaml]: why is there extra space introduced?

File: stable/openebs/templates/deployment-ndm-operator.yaml [hunk 3]
```diff
@@ -56,10 +58,10 @@ spec:
...
 {{ toYaml .Values.ndmOperator.nodeSelector | indent 8 }}
 {{- end }}
 {{- if .Values.ndmOperator.tolerations }}
-      tolerations:
+     tolerations:
```
Comment (kmova) [inline stable/openebs/templates/deployment-ndm-operator.yaml]: same as above. is this change required.

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Review by kmova — APPROVED** [review]
/lgtm
/approved
